Samples/DistributionSample/DistributionSample/UpdateUtil.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import SentryDistribution
+import UIKit
+
+enum UpdateUtil {
+  @MainActor
+  static func checkForUpdates() {
+    let params = CheckForUpdateParams(accessToken: Constants.accessToken, organization: Constants.organization, project: Constants.project)
+    Updater.shared.checkForUpdate(params: params) { result in
```

Comment:
`h`: The SentrySDK's interface is using `SentrySDK.<method>` without having to access a singleton first. We should keep that pattern.